Fix for 22177995 - /dev/console and /dev/mslog redirection getting garbled 
in legacy locale on s11

--- xterm-410/main.c
+++ xterm-410/main.c
@@ -2937,6 +2937,15 @@ if (command_to_exec_with_luit == NULL)
 	    SysError(ERROR_LUMALLOC);
 
 	x_appendargv(command_to_exec_with_luit, split_filter);
+
+        if (Console) {
+            char *encoding_opt1[2];
+            int pos = 0;
+            encoding_opt1[pos++] = "-C";
+            encoding_opt1[pos] = 0;
+            x_appendargv(command_to_exec_with_luit, encoding_opt1);
+        }
+
 	if (count_using) {
 	    char *encoding_opt[4];
 	    encoding_opt[0] = x_strdup("-encoding");
